diff -ru ../sqlwork-5.0-orig/Makefile.am ./Makefile.am
--- ../sqlwork-5.0-orig/Makefile.am	Tue Feb 15 17:58:15 2000
+++ ./Makefile.am	Fri Aug 25 13:16:51 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = intl po src
+SUBDIRS = intl po gtkeditor src
 
 install-data-local:
 	@$(NORMAL_INSTALL)
diff -ru ../sqlwork-5.0-orig/src/Makefile.am ./src/Makefile.am
--- ../sqlwork-5.0-orig/src/Makefile.am	Fri Jul 28 04:29:18 2000
+++ ./src/Makefile.am	Fri Aug 25 13:44:57 2000
@@ -29,6 +29,7 @@
 
 DB_SOURCES = @DBSRC@
 DB_OBJS = $(subst .c,.o,$(filter %.c, $(DB_SOURCES)))
+GTKEDITOR = -L../gtkeditor -lgtkeditor
 
 INCLUDES = \
 	-I$(top_srcdir)/intl \
@@ -48,6 +49,6 @@
 	bookmarksig.c bookmarksig.h oscan.l $(DB_SOURCES)
 
 sqlwork_DEPENDENCIES = $(DB_OBJS)
-sqlwork_LDADD = $(DB_OBJS) @DBLIBS@ @GTK_LIBS@ $(LEXLIB) $(INTLLIBS)
+sqlwork_LDADD = $(DB_OBJS) @DBLIBS@ @GTK_LIBS@ $(GTKEDITOR) $(LEXLIB) $(INTLLIBS)
 
 CLEANFILES = dblist.c
diff -ru ../sqlwork-5.0-orig/src/sqlworkwin.c ./src/sqlworkwin.c
--- ../sqlwork-5.0-orig/src/sqlworkwin.c	Tue May  9 15:32:43 2000
+++ ./src/sqlworkwin.c	Fri Aug 25 13:17:59 2000
@@ -1272,7 +1272,9 @@
   gtk_widget_set_usize (scrolledwindow1, -2, 160);
   gtk_scrolled_window_set_policy (GTK_SCROLLED_WINDOW (scrolledwindow1), GTK_POLICY_NEVER, GTK_POLICY_ALWAYS);
 
-  sql_query = gtk_text_new (NULL, NULL);
+/*  sql_query = gtk_text_new (NULL, NULL);*/
+  sql_query = gtk_editor_new(NULL, NULL);
+  install_pat(GTK_EDITOR(sql_query), work_pane);
   gtk_widget_ref (sql_query);
   gtk_object_set_data_full (GTK_OBJECT (work_pane), "sql_query", sql_query,
                             (GtkDestroyNotify) gtk_widget_unref);
diff -ru ../sqlwork-5.0-orig/src/sqlworkwin.h ./src/sqlworkwin.h
--- ../sqlwork-5.0-orig/src/sqlworkwin.h	Tue May  9 15:32:43 2000
+++ ./src/sqlworkwin.h	Fri Aug 25 13:26:12 2000
@@ -24,6 +24,10 @@
  * change the formatting or indentation
  */
 
+#include "../gtkeditor/gtkeditor.h"
+
+void install_pat (GtkEditor *, GtkWidget *);
+
 GtkWidget* create_main_window (void);
 GtkWidget* create_login_window (void);
 GtkWidget* create_work_pane (void);
Only in .: sw50_gtkeditor.patch
